lib/game.rb

```diff
@@ -0,0 +1,55 @@
+class Game
+  attr_reader :game_id,
+              :season,
+              :type,
+              :date_time,
+              :away_team_id,
+              :home_team_id,
+              :away_goals,
+              :home_goals,
+              :venue,
+              :venue_link
```

Comment:
You have a lot of attr_readers here, I'd recommend trying to separate this a bit.

Reply:
What is the downfall to having too many attr_readers?